assembly/src/parsers/mod.rs

```diff
@@ -152,3 +152,32 @@ fn parse_element_param(op: &Token, param_idx: usize) -> Result<BaseElement, Asse
 
     Ok(BaseElement::new(result))
 }
+
+/// This is a helper function that validates the length of the assembly io instruction and returns
```

Comment:
I think this can be used not only for IO instructions. Maybe remove the "io" part?